Esteban-Salas-Posada/tarea2/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Formulario</title>
+</head>
+<body>
+    <form action="" class="form">
+
+        <fieldset class="form__personaldata">
+            <legend class="form__legend">Personal information</legend>
+                <section>
+                    <label for="name">* Type your name: </label>
+                    <input type="text" name="name" id="name" tap="1" placeholder="Your name here" required>
+                </section>
+                <br>
```

Comment:
@esteban-salas , avoid the use of BR , the layout should come from CSS properties int he section tags